sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -234,6 +235,17 @@ export function getFileNameAsCurrentTimestamp(): string {
   return dayjs(new Date()).tz(PST_TIMEZONE).format(TIMESTAMP_CONTINUOUS_FORMAT);
 }
 
+/**
+ * Return a PST timestamp with date and time in customized continuous format
+ * mainly used to append in filename.
+ * @returns timestamp.
+ */
+export function getFileNameAsCustomizedCurrentTimestamp(): string {
+  return dayjs(new Date())
+    .tz(PST_TIMEZONE)
```

Comment:
Will this be PST? I couldn't find in the ticket whether it is PST or UTC.

Reply:
I used the one we were using it previously for the conversion.
![image](https://github.com/bcgov/SIMS/assets/62901416/88cfa896-7557-454c-8a13-2d65893e5b16)

Reply:
@guru-aot the previously existing method was used for the purpose ministry user downloading the report from ministry portal. 

I am not sure if that is same with integration files. 

In case if not, my comment https://github.com/bcgov/SIMS/pull/2785#discussion_r1480597740 can be ignored.

Reply:
@guru-aot maybe we should add a note to the ticket about it. 
I have just a guess that Ministry would prefer the timestamp in PST also but a confirmation from @JasonCTang would be helpful. Maybe just ask about with timezone the file in the `SIMS\IN` folder will be.